src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -290,6 +294,8 @@ public static void main(String[] args) throws Exception {
                     terminals,
                     Phase.Arrival.REGULAR);
 
+      jsonMetricsHelper.buildTestConfig(nodes.size(),numWarehouses, numDBConnections, warmupTime, time, wrkld.getMaxRetriesPerTransaction());
```

Comment:
nit -- lets respect some line limit here. for new code, lets stick to the same 100 char line limit we have in c++ code (though i dont know if we have formalized anything for java somewhere, might be worth looking for a rule we could copy from platform code..)